kernel/Kconfig

```diff
@@ -479,6 +479,15 @@ config NUM_PIPE_ASYNC_MSGS
 	  Setting this option to 0 disables support for asynchronous
 	  pipe messages.
 
+config MEMPOOL
+	bool "Memory pools (k_mem_pool)"
+	default y
+	help
+	  If the mempool is not used, this can be disabled to save space. The
+	  mempool is used by a number of internal modules, so keep this in mind
```

Comment:
Well, shouldn't you have a `depends on` statement somewhere then?

Reply:
depends on what?